dietpi/dietpi-software

```diff
@@ -4332,6 +4339,223 @@ _EOF_
 			G_DIETPI-NOTIFY 0 'microblog.pub installation finished. In order to complete the setup, please run "microblog-pub configure" in a new bash session. Append the port ":8007" to the domain when being asked for it if you do not use a reverse proxy.'
 		fi
 
+		if To_Install 210 # lemmy
+		then
+			aDEPS=('make' 'gcc' 'libc6-dev' 'pkg-config' 'libssl-dev' 'protobuf-compiler' 'libprotobuf-dev' 'libpq-dev')
+
+			# This script creates the lemmy user, a dedicated postgres
+			# db in the dietpi userspace, the lemmy backend, the lemmy
+			# frontend, and apparently the pict-rs image server too
+			# (that last one is NOT clear).
+
+			## Install up-to-date rustup (cargo version in debian is too old)
+			local lemm_user='lemmy'
+			local lemm_home="/home/$lemm_user"
+			local lemm_server_path=$lemm_home/.cargo/bin
+			# User
+			##Create_User -G dialout,gpio,i2c -d "$lemm_home" "$lemm_user"
+			G_EXEC mkdir -p "$lemm_home"
+			Create_User -d "$lemm_home" "$lemm_user"
+			G_EXEC chown "$lemm_user":"$lemm_user" -R "$lemm_home"
+
+			##G_EXEC sudo su -u $lemm_user mkdir -p $lemm_bin
+
+			## Install Rust, cargo
+			## We assume users will want to upload images, so we enable pictrs
+			G_DIETPI-NOTIFY 0 '[ INFO ] lemmy backend installation started'
```

Comment:
```suggestion
			G_DIETPI-NOTIFY 2 'Lemmy backend installation started'
```
- `G_DIETPI-NOTIFY 0` is for `[  OK  ]` messages
- `G_DIETPI-NOTIFY 1` is for `[FAILED]` messages
- `G_DIETPI-NOTIFY 2` is for `[ INFO ]` messages
- The prefix is added automatically